src/sonic-bgpcfgd/bgpmon/bgpmon.py

```diff
@@ -43,10 +43,9 @@ def __init__(self):
         self.new_peer_l = set()
         self.new_peer_state = {}
         self.cached_timestamp = 0
-        self.db = swsssdk.SonicV2Connector()
+        self.db = swsscommon.SonicV2Connector()
         self.db.connect(self.db.STATE_DB, False)
-        client = self.db.get_redis_client(self.db.STATE_DB)
-        self.pipe = client.pipeline()
+        self.pipe = self.db.get_redis_client(self.db.STATE_DB)
```

Comment:
Remove pipe because in flush_pipe(self, data) has following code:
            if value is None:
                # delete case
                self.pipe.delete(key)
            else:
                # Add or Modify case
                self.db.hmset(self.pipe.STATE_DB. key, value)


This is because swsscommon not using pyredis, and self.db.get_redis_client(self.db.STATE_DB) will return StrictRedis. so the StrictRedis.pipeline() method does not exist in sonic-swss-common.

Also another reason is the DBConnector only has following 1 method in swsscommon.py:

    def hmset(self, multiHash):
        return _swsscommon.DBConnector_hmset(self, multiHash)